src/viewport/viewport.model.ts

```diff
@@ -1,23 +1,14 @@
-export interface ViewportSize {
-	width: number;
-	height: number;
-}
+import { Dictionary } from "../internal/internal.model";
 
-export enum ViewportSizeType {
-	xsmall = 0,
-	small = 1,
-	medium = 2,
-	large = 3,
-	xlarge = 4,
-	xxlarge = 5,
-	xxlarge1 = 6
-}
-
-export interface ViewportSizeTypeInfo {
-	type: ViewportSizeType;
-	name: string;
-	widthThreshold: number;
-}
+export const ViewportSizeType = {
```

Comment:
as we had discussed the values of this should remain as they were (so indices instead of widths)